src/test/ui/consts/const_in_pattern/issue-65466.rs

```diff
@@ -1,3 +1,7 @@
+// FIXME: This still ICEs.
```

Comment:
ah, boo hoo. 😢 

(still, if ICE today, then ICE tomorrow, as I'm sure someone on the team has said at some point...)